packages/rancher-monitoring/package.yaml

```diff
@@ -1,5 +1,5 @@
 url:  https://github.com/prometheus-community/helm-charts/releases/download/kube-prometheus-stack-9.4.2/kube-prometheus-stack-9.4.2.tgz
-packageVersion: 02
+packageVersion: 03
```

Comment:
packageVersion change only necessary if this is going out after 2.5.4, not necessary if this is going into 2.5.4